### PR TITLE
gitignore fixture

### PR DESCRIPTION
**Pull request by chazu**
Git ignore things.

### CODE REVIEW

File: lib/jarbs/project_generator.rb [hunk 1]
```diff
@@ -33,6 +33,11 @@ def generate
 
       write_package(manifest, @name)
       NodeBuild.new(nil).npm_install(@name)
+      install_handler
+    end
+
+    def install_handler
```
Comment (plukevdh) [inline lib/jarbs/project_generator.rb]: maybe change the name to `install_gitignore`?

**Comment by plukevdh** [comment]
Very much dig

File: lib/jarbs/project_generator.rb (hunk 1 above)
Comment (chazu) [inline lib/jarbs/project_generator.rb]: Ah, I thought install_handler meant 'the handler for the install [of fixtures]'. Derp. >.<

File: lib/jarbs/function_generator.rb [hunk 2]
```diff
@@ -40,7 +40,7 @@ def install_manifest
       write_package(package_manifest, @function.source_path)
     end
 
-    def install_handler
+    def install_gitignore
```
Comment (plukevdh) [inline lib/jarbs/function_generator.rb]: haha, you changed the wrong one

**Pull request closed without merging**